packages/components/src/text/text.tsx

```diff
@@ -131,6 +132,12 @@ const Base = styled(BaseText, {
         minWidth: 0,
       },
     },
+
+    select: {
+      false: {
+        userSelect: 'none',
+      },
```

Comment:
So if `select` is `undefined` is the same output that writing this `select={true}` or just pass the `select` prop. Isn't that a bit ambiguous? Since it's selectable by default maybe it is best to have the type as `select?: false`. Wdyt? Makes sense?

Reply:
Like that 👍, https://github.com/status-im/status-web/pull/492/commits/71abb3630eb835bddb5bc74c7fa2d0cda5b053f9.